src/common.speech/CognitiveTokenAuthentication.ts

```diff
@@ -24,10 +24,10 @@ export class CognitiveTokenAuthentication implements IAuthentication {
     }
 
     public fetch = (authFetchEventId: string): Promise<AuthInfo> => {
-        return this.privFetchCallback(authFetchEventId).then((token: string) => new AuthInfo(AuthHeader, token));
+        return this.privFetchCallback(authFetchEventId).then((token: string) => new AuthInfo(AuthHeader, "bearer " + token));
```

Comment:
>"bearer " [](http://example.com/codeflow?start=105&length=9)

Is it common practice to duplicate hard coded strings like this?

in our other API surfaces, these would all be pulled out the a webheaders and cookies .h file and defined as a reusable constant.

Reply:
I think you're correct, this should be a class level constant. Will update the other PR to update this.